corehq/apps/indicators/utils.py

```diff
@@ -14,6 +16,33 @@ def get_indicator_config():
         return doc.get('namespaces', {})
 
 
+def set_domain_namespace_entry(domain, entry):
+    with CriticalSection(['udpate-indicator-configuration-doc']):
+        try:
+            doc = get_db().open_doc('INDICATOR_CONFIGURATION')
+        except ResourceNotFound:
+            doc = {
+                '_id': 'INDICATOR_CONFIGURATION',
+            }
+
+        if 'namespaces' not in doc:
+            doc['namespaces'] = {}
+
+        doc['namespaces'][domain] = entry
+        get_db().save_doc(doc)
+        get_indicator_config.clear()
+
+
+@unit_testing_only
+def delete_indicator_doc():
```

Comment:
What do you think of moving this to a file in the `tests` package since this is for unit tests only? In general non-test modules should not contain test functions.